torch/distributed/_tensor/placement_types.py

```diff
@@ -199,8 +199,8 @@ def _reduce_shard_tensor(
             )
             tensor = torch.cat(scattered_list, dim=self.dim)
 
-        output = mesh.reduce_scatter(
-            tensor, op=reduce_op, mesh_dim=mesh_dim, scatter_dim=self.dim
+        output = funcol.reduce_scatter_tensor(
+            tensor, reduce_op.name, scatter_dim=self.dim, group=(mesh, mesh_dim)
```

Comment:
Should we support reduce_op directly as well in the python wrappers of funcol?

Reply:
yeah that's a good point.. I can give it a try in a follow up PR